features/step_definitions/supplier_utils.rb

```diff
@@ -0,0 +1,138 @@
+#Utils
+
+# suppliers
+def create_supplier (supplier_id, supplier_name, supplier_description, supplier_contactName, supplier_email, supplier_postcode, supplier_dunsnumber)
```

Comment:
We have [a function in our api helpers](https://github.com/alphagov/digitalmarketplace-functional-tests/blob/master/features/support/api_helpers.rb#L220) to create suppliers. At the moment it's not paramaterized, so I think you can only create the same supplier. You may be able to update it so you can create different suppliers.